src/style-auro-ld.scss

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// general WCSS imports
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/utilityMixins/focusVisible";
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation";
+
+// Import Auro SCSS variables
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables";
+
+// import button styles into element namespaced wrapper
+auro-button-light {
+  @import './style-auro.scss';
+}
+
+// light DOM support for hover states
+auro-button-light:not(.is-touching) {
+  .auro-button {
+    &:hover {
+      @include auro_transition(all, 150ms, ease-in-out);
+      
+      background-color: var(--auro-color-ui-hover-on-light);
+      border: 1px solid var(--auro-color-ui-hover-on-light);
+    }
+
+    &--secondary {
+      &:hover {
+        @include auro_transition();
+
+        background-color: var(--auro-color-background-lighter);
+        border: 1px solid var(--auro-color-ui-default-on-light);
+      }
+    }
+
+    &--tertiary {
+      &:hover {
+        @include auro_transition();
+
+        background-color: var(--auro-color-background-lighter);
+        border-color: transparent;
+      }
+    }
+
+    &Ondark {
+      &:hover {
+        @include auro_transition(all, 150ms, ease-in-out);
+        
+        background-color: var(--auro-color-ui-hover-on-dark);
+        border: none;
+      }
+
+      &--secondary {
+        &:hover {
+          background-color: rgba(99, 95, 95, 0.15);
+        }
+      }
+
+      &--tertiary {
+        &:hover {
+          background-color: rgba(255, 255, 255, 0.15);
+        }
+      }
+    }
+  }
+}
+
+// light DOM support for focus-visible
+auro-button-light .focus-visible.auro-button {
```

Comment:
2 things. 

Is this right? In the light DOM, isn't the focus-visible selector on the auro-button-light element? And then .auro-button would be a descendent selector?

Reply:
This selector resolves to the following:
`
<auro-button-light>
  …
    <element class="focus-visible auro-button">
`
Example of placement in the DOM:
`
<auro-button-light tertiary content="Tertiary default state">     
      <!-- styles -->
      <button class="util_insetLg--squish auro-button auro-button--tertiary focus-visible">
        <!---->Tertiary default state<!---->
      </button>
</auro-button-light>
`

Should we evaluate moving focus-visible up to auro-button-light?

Reply:
Oops, that's my bad. I forgot that the focus-visible appears on the button element in the light DOM version.